testing/bdd/templates/padding_data_type.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" ?>
+<tt:tt ebuttp:sequenceIdentifier="testSequence001" ebuttp:sequenceNumber="1" ttp:clockMode="local" ttp:timeBase="clock" xml:lang="en-GB" xmlns:ebuttm="urn:ebu:tt:metadata" xmlns:ebuttp="urn:ebu:tt:parameters" xmlns:tt="http://www.w3.org/ns/ttml" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:xml="http://www.w3.org/XML/1998/namespace">
+  <tt:head>
+    <tt:metadata>
+      <ebuttm:documentMetadata/>
+    </tt:metadata>
+    <tt:styling>
+{% if test_padding_syntax %}
+    <tt:style xml:id="ID001" tts:padding="{{value1}} {{value2}} {{value3}} {{value4}}"/>
+{% else %}
+    <tt:style xml:id="ID001" {{style_padding}}/>
+{% endif %}
+    </tt:styling>
+    <tt:layout>
+      <tt:region xml:id="ID002" tts:origin="0% 0%" tts:extent="100% 100%" {{region_padding}}/>
```

Comment:
Because I test the permutations of the last 2 scenarios only on the style attribute, I can test them on the region attribute too.